gulpfile.js/tasks/browserify.js

```diff
@@ -38,6 +38,11 @@ var browserifyTask = function (callback, devMode) {
       bundleConfig = _.omit(bundleConfig, ['external', 'require'])
     }
 
+    if (!isDev) {
+      // stop absolute urls appearing in prod browserify'd bundles
+      _.extend(bundleConfig, {fullPaths: false, plugin: collapse})
```

Comment:
`fullPaths` doesn't seem to do anything for me. I might be missing something but it feels like that should be all we need?